### PR TITLE
Add profile tab

### PR DESCRIPTION
**Pull request by zeraye**
I implemented profile tab. Currently: `name`, `lastName`, `email` and `password` are only changed locally (backend is not implemented for this feature). You can see details in the video below:

https://github.com/flatly-pw/mobile-app/assets/53649257/f5a21846-5a0c-41b5-a948-9c9371a74165

Note: In this example video there is `dolar` as a currency. After recording the video, this name was fixed to `dollar`.

### CODE REVIEW

File: features/ProfileTab/PreferencesScreen/Currency/CurrencyDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { useContext, useState } from "react";
+
+import SettingsContext from "../../../../contexts/SettingsContext";
+import Currency from "../../../../types/Currency";
+
+const data: {
+  label: string;
+  value: Currency;
+}[] = [
+  { label: "Dolar", value: "USD" },
```
Comment (kemeyr) [inline features/ProfileTab/PreferencesScreen/Currency/CurrencyDropdown.tsx:10]: label: "US Dollar" would be the best option (many countries use their own dollars)

File: features/ProfileTab/PreferencesScreen/CurrencyDropdown/CurrencyDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { Surface, Text } from "react-native-paper";
+
+import SettingsContext from "../../../../contexts/SettingsContext";
+import Currency from "../../../../types/Currency";
+
+const data: {
+  label: string;
+  value: Currency;
+}[] = [
+  { label: "Dollar", value: "USD" },
```
Comment (kemeyr) [inline features/ProfileTab/PreferencesScreen/CurrencyDropdown/CurrencyDropdown.tsx]: label: "US Dollar" as above

File: features/ProfileTab/PreferencesScreen/Currency/CurrencyDropdown.tsx (hunk 1 above)
Comment (zeraye) [reply features/ProfileTab/PreferencesScreen/Currency/CurrencyDropdown.tsx:10]: Fixed in [fd48dd3](https://github.com/flatly-pw/mobile-app/pull/13/commits/fd48dd317fdca05be37b7145fffa014f31b955d2)

File: features/ProfileTab/PreferencesScreen/CurrencyDropdown/CurrencyDropdown.tsx (hunk 2 above)
Comment (zeraye) [reply features/ProfileTab/PreferencesScreen/CurrencyDropdown/CurrencyDropdown.tsx]: Fixed in [fd48dd3](https://github.com/flatly-pw/mobile-app/pull/13/commits/fd48dd317fdca05be37b7145fffa014f31b955d2)

**Review by kemeyr — APPROVED** [review]
No issues found

**Pull request merged**